<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTimeCard</class>
 <widget class="QFrame" name="CTimeCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1390</width>
    <height>818</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>计时卡</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image:url(D:/WinParkUI/debug/Image/NewIcon/CommonBigBG-normal.jpg);</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>1301</width>
     <height>731</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
   <widget class="QTableWidget" name="tableTime">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1301</width>
      <height>221</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <column>
     <property name="text">
      <string>卡号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>卡类型</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>收费标准</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>卡片自编号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>卡状态</string>
     </property>
    </column>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>1301</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>最近进出记录</string>
     </attribute>
     <widget class="QTableWidget" name="tableAccess">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>841</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(none);</string>
      </property>
      <column>
       <property name="text">
        <string>进入通道</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>进入时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>离开通道</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>离开时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>记录ID</string>
       </property>
      </column>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>0</y>
        <width>211</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(none);</string>
      </property>
      <property name="title">
       <string>图像1</string>
      </property>
      <widget class="QLabel" name="lblImg1">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>16</y>
         <width>201</width>
         <height>191</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_14">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>0</y>
        <width>211</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(none);</string>
      </property>
      <property name="title">
       <string>图像2</string>
      </property>
      <widget class="QLabel" name="lblImg2">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>16</y>
         <width>201</width>
         <height>191</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_15">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>210</y>
        <width>211</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(none);</string>
      </property>
      <property name="title">
       <string>图像3</string>
      </property>
      <widget class="QLabel" name="lblImg3">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>16</y>
         <width>201</width>
         <height>191</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_16">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>210</y>
        <width>211</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(none);</string>
      </property>
      <property name="title">
       <string>图像4</string>
      </property>
      <widget class="QLabel" name="lblImg4">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>16</y>
         <width>201</width>
         <height>191</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>出入权限</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>1200</x>
        <y>400</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableEntranceRight">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1271</width>
        <height>371</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>通道名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>进入时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>离开时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="lblCardNo">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>400</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnMinimalize">
    <property name="geometry">
     <rect>
      <x>683</x>
      <y>700</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>5</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);
color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblClose">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>7</y>
     <width>30</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
